packages/twenty-front/src/modules/object-metadata/utils/getDefaultValueForBackend.ts

```diff
@@ -0,0 +1,19 @@
+import { FieldCurrencyValue } from '@/object-record/record-field/types/FieldMetadata';
+import { FieldMetadataType } from '~/generated-metadata/graphql';
+
+export const getDefaultValueForBackend = (
```

Comment:
Reading the code, I'm not sure about it, i think this should go to the BE
I wonder if the backend should actually do the work and expect a different kind of defaultValue per FieldType.
If the fieldType is CurrencyCode, we don't care about uuid(), now() functions indeed, we just want currencyCode (string) + amountMicros (int)

Reply:
I'm in favor of moving forward with the current implementation and take the time to refactor next week, should be quite light